prettyping

```diff
@@ -813,7 +813,7 @@ BEGIN {
 
 		rtt = $0
 		sub(/.* time=/, "", rtt)
-		rtt = int(rtt)
+		rtt = rtt + 0
```

Comment:
```suggestion
		rtt = rtt + 0.0
```

Should we add `0.0` instead of `0`, just to make it explicit we are casting/converting this to a float instead of an int? What do you think?